<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewNodeDialog</class>
 <widget class="QDialog" name="NewNodeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Node</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ui_leName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ui_leTitle"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QRadioButton" name="ui_rbFile">
         <property name="text">
          <string>File</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="value" stdset="0">
          <string>F</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ui_bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ui_rbDir">
         <property name="text">
          <string>Directory</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="value" stdset="0">
          <string>D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ui_bgType</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ui_rbResource">
         <property name="text">
          <string>Resource</string>
         </property>
         <property name="value" stdset="0">
          <string>R</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ui_bgType</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Resource Content</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ui_leResContent">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewNodeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewNodeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_rbResource</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_rbResource</sender>
   <signal>toggled(bool)</signal>
   <receiver>ui_leResContent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="ui_bgType"/>
 </buttongroups>
</ui>
